housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java

```diff
@@ -76,8 +76,9 @@ public void postConstruct() {
         .put("spring.datasource.max-wait", 10000)
         .put("spring.datasource.max-active", 2)
         .put("spring.datasource.test-on-borrow", true)
-        .put("spring.datasource.schema", "${housekeeping.db-init-script:classpath:/schema.sql}")
-        .put("housekeeping.h2.database", "${instance.home}/data/${instance.name}/housekeeping")
+        .put("spring.datasource.schema", "${housekeeping.db-init-script: }")
+        .put("housekeeping.h2.database",
+            "${instance.home}/data/${instance.name}/${housekeeping.schema-name:housekeeping}")
```

Comment:
I changed this line so when H2 is used, a database with the file with the same name as the schema name is created. I tested it for schema `custom_schema`, and instead of getting `/data/housekeeping.mv.db` (what I got with the latest released version of HK), I got `/data/custom_schema.mv.db`.